config/default.json

```diff
@@ -0,0 +1,12 @@
+{
+	"curator": {
+		"languages": {
+			"rb": "Ruby",
+			"cs": "C#",
+			"js": "JavaScript"
+		},
+		"commands": [
+			{"init": "initialize cwd with _templates folder, stream-specific subfolders, and template md files"}
```

Comment:
yeah i think we added that on the fly and actually arent using it currently,we  should think about the structure